source/includes/app.md

```diff
@@ -46,30 +46,37 @@ app.elements.destroy(button);
 
 The `elements` object can be used to query all scenes for an element, create elements, or destroy them.
 
-## SAI object
+## SAI object (Preview)
```

Comment:
I think it's too late for this one, but in the future, I think all preview apis should have the `-preview` suffix in the import, i.e. `const sai = require('sai-preview');`.